src/showingpreviously/cinemas/isle_of_bute_discovery_centre_cinema.py

```diff
@@ -0,0 +1,95 @@
+import re, lxml, datetime
+
+from bs4 import BeautifulSoup
+from typing import Tuple
+
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE
+
+CHAIN = Chain('Isle of Bute Discovery Centre')
+CINEMA = Cinema('Isle of Bute Discovery Centre Cinema', UK_TIMEZONE)
+SCREEN = Screen('Screen 1')
+
+FEED_URL = 'http://discoverycentrecinema.blogspot.com/feeds/posts/default'
+FILM_TITLE_REGEX = re.compile('(?P<title>.*) (?P<rating>\(..*\))')
+
+
+class InvalidRowException(Exception):
+    pass
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+class IsleOfButeDiscoveryCentreCinema(ChainArchiver):
+    def get_showings(self) -> [Showing]:
+        showings = []
+        feed_contents = get_response(FEED_URL).text
+        feed_soup = BeautifulSoup(feed_contents, features='xml')
+        html_contents = feed_soup.find('entry').find('content', type='html').contents
+        html_soup = BeautifulSoup(html_contents[0], features='html.parser')
+        listings_table = html_soup.find('table')
+        for i, row in enumerate(listings_table.find_all('tr')):
+            if i == 0:
+                continue # skip the 'Film listings for <Month> row'
```

Comment:
```suggestion
                continue  # skip the 'Film listings for <Month> row'
```
Two spaces before the comment please